projects/plugins/jetpack/extensions/blocks/podcast-player/attributes.js

```diff
@@ -17,6 +17,10 @@ export default {
 		type: 'string',
 		validator: isUrl,
 	},
+	selectedEpisodes: {
+		type: 'array',
+		default: [],
+	},
```

Comment:
`selectedEpisodes` is an array of episode objects. In a separate, future PR, we will cache episode details here, in case the selected episodes are ever removed from the RSS feed.

Reply:
What do you think about swapping to one attribute and, for now, have the GUID be the only property of each object in this array?

```js
[ { guid: '123435324234abc' }, { guid: '123435324234def' }, { guid: '123435324234ghi' }, ... ]
```

We will need the GUIDs in this array anyway, and we could construct the GUID list on the fly if we needed it for convenience.

Reply:
Ignore me! I've just seen that's exactly what's happening.

Reply:
Probably we'll have to consolidate the podcast-player.js component with this block. Currently, the component stores the details of the episodes in the `tracks` property.
<img src="https://user-images.githubusercontent.com/77539/105345074-8b131a00-5bc2-11eb-9c82-11fe6ad782b6.png" width="600px" />